site/_posts/2017-09-21-finalizers-can-be-interrupted-from-time-to-time.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Finalizers can be interrupted from time to time
 author: jrafanie
-date: 2017-09-21 15:36:39 UTC
+date: 2017-09-28 19:05:09 UTC
```

Comment:
As requested @Fryguy

Reply:
wait whut why not rename the file...?

Reply:
@cybette suggested just changing it here.  I can rename the file too. I wasn't sure how links to the blog article would be affected, such as: http://www.manageiq.org/blog/2017/09/finalizers-can-be-interrupted-from-time-to-time/

Let me know what we want to do.

Reply:
Note, those links don't include the day, only the year and month so maybe we'd be fine.

Reply:
yeah it doesn't really matter, whatever :)